### PR TITLE
ISPN-15736 Log broadcast queries and loading

### PR DESCRIPTION
**Pull request by fax4ever**
https://issues.redhat.com/browse/ISPN-15736
https://issues.redhat.com/browse/JDG-6871

### CODE REVIEW

File: query/src/main/java/org/infinispan/query/logging/Log.java [hunk 1]
```diff
@@ -211,4 +212,21 @@ public interface Log extends BasicLogger {
    ParsingException booleanKnnPredicates();
 
    // !!!!!! When adding anything new here please check the last used id in org.infinispan.query.core.impl.Log !!!!!!
+
+   @LogMessage(level = TRACE)
```
Comment (tristantarrant) [inline query/src/main/java/org/infinispan/query/logging/Log.java:216]: Trace and debug messages don't go in Log because they should not be i18n-ized

**Review by tristantarrant — CHANGES_REQUESTED** [review]

File: query/src/main/java/org/infinispan/query/logging/Log.java (hunk 1 above)
Comment (fax4ever) [reply query/src/main/java/org/infinispan/query/logging/Log.java:216]: I always forget it :( Pushing the fix!

**Comment by fax4ever** [comment]
Thank @tristantarrant for the review. I applied the change.

**Comment by tristantarrant** [comment]
Thanks

**Comment by fax4ever** [comment]
Thanks Tristan!

**Pull request merged**